clouddriver-docker/src/main/groovy/com/netflix/spinnaker/clouddriver/docker/registry/config/DockerRegistryConfigurationProperties.groovy

```diff
@@ -37,6 +37,10 @@ class DockerRegistryConfigurationProperties {
     String address
     // How many threads to cache all provided repos on. Really only useful if you have a ton of repos.
     int cacheThreads
+    // Timeout time in milliseconds for this repository. Default is 60,000 (1 minute).
+    long clientTimeoutMillis
+    // Paginate size for the docker repository /_catalog endpoint. Default is 100
+    int paginateSize
```

Comment:
@tomaslin I moved clientTimeout into clientTimeoutMillis to per-account configuration. Getting the beans to work was too much of a pain, as the autowired dependencies aren't available until after the constructor runs.